Packs/CIRCL/ReleaseNotes/1_0_27.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### CIRCL CVE Search
+
+- Deprecated cve_latest command - Use NVD Feed 2.0 instead.
+- Updated the cve command to support the new 5.1 format.
```

Comment:
Use formatting around the commands - https://xsoar.pan.dev/docs/documentation/release-notes#entity-styling